desktop/server.js

```diff
@@ -21,10 +21,12 @@ if (NO_SERVER) {
     if (jsonStats.errors.length > 0) {
       throw jsonStats.errors.join('\n')
     }
+
+    console.log(stats)
+
     if (jsonStats.warnings.length > 0) {
```

Comment:
Are there any unused files after your PR? I wonder if we should add a test to CI to avoid adding new ones.